tests/Unit/Sabre/AbstractBearerTest.php

```diff
@@ -68,27 +65,23 @@ function testRequireAuth() {
 			'Bearer realm="writing unittests on a saturday night"',
 			$response->getHeader('WWW-Authenticate')
 		);
-
 	}
-
 }
 
-
-class AbstractBearerMock extends AbstractBearer {
-
+class AbstractBearerMock extends AbstractBearer
```

Comment:
code style :boom:

Reply:
:( I pasted from the Sabre Dav code.

Reply:
it's okay - we add php-csfixer in a followup pr